src/test/java/de/medizininformatikinitiative/torch/RedactTest.java

```diff
@@ -44,8 +44,11 @@ public void testObservationLab(String resource) throws IOException {
     }
 
     @ParameterizedTest
-    @ValueSource(strings = {"Observation-mii-exa-test-data-patient-1-vitalstatus-1.json"})
-    public void testObservationVitalStauts(String resource) throws IOException {
+    @ValueSource(strings = {
+            "Observation-mii-exa-test-data-patient-1-vitalstatus-1.json",
+            "Observation-mii-exa-test-data-patient-1-vitalstatus-1-identifier.json"
+    })
+    public void testObservationVitalStatus(String resource) throws IOException {
```

Comment:
The test name should reflect what you test here. I suspect you test the redaction of the Reference complex type.